cyber/tools/cyber_monitor/renderable_message.h

```diff
@@ -82,6 +84,7 @@ class RenderableMessage {
   int page_item_count_;
   RenderableMessage* parent_;
   double frame_ratio_;
+  double band_width_;
```

Comment:
Set init value here, remove from constructor.

Reply:
Other data members are initialized by constructors. Will direct initialization be inconsistent?